src/include/migraphx/op/pooling.hpp

```diff
@@ -42,16 +42,48 @@ namespace op {
 
 struct pooling
 {
-    pooling_mode mode                = {pooling_mode::average};
+    pooling_mode mode = {pooling_mode::average};
+
+    // The amount each dimension is padded, both before and after.  Can be
+    // either the same size as lengths, or twice as long.  When padding
+    // is double the length of lengths, the first n values are read as
+    // pre-padding for each dimension and the last n values are post-padding.
+    // (The latter is necessary if the total padding desired is an odd number)
+    //
+    //
     std::vector<std::size_t> padding = {0, 0};
-    std::vector<std::size_t> stride  = {1, 1};
+
+    // Size of stride to take from one placement of the pooling kernel to the next.
+    // This is distinct from the strides used by the shape class.  If set the same as lengths, the
+    // kernel tiles over the input with no gaps or overlaps. Must be the same size as lengths.
```

Comment:
```suggestion
    // Striding for the pooling kernel
```
Too much information. Developer can look at any outside pooling description to understand this.